### PR TITLE
ARO-8392: Added mdsd config to ignition

### PR DESCRIPTION
**Pull request by rna-afk**
Generating the files needed for mdsd work using the code in
the wrapper that was taken from the ARO fork and adding 
to ignition.

### CODE REVIEW

File: pkg/installer/mdsd/util.go [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+import (
+	"bytes"
+	"html/template"
+	"io"
+	"path"
+	"path/filepath"
+	"strings"
+
+	ignutil "github.com/coreos/ignition/v2/config/util"
+	igntypes "github.com/coreos/ignition/v2/config/v3_2/types"
```
Comment (zaneb) [inline pkg/installer/mdsd/util.go:12]: If you add these names to https://github.com/openshift/installer-aro-wrapper/blob/main/hack/validate-imports/allowed-import-names.yaml then the validate job will be happy.

File: pkg/installer/mdsd/util.go [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+	"github.com/openshift/installer/pkg/asset/ignition"
+)
+
+func AddStorageFiles(config *igntypes.Config, base string, uri string, templateData interface{}) (err error) {
```
Comment (zaneb) [inline pkg/installer/mdsd/util.go:16]: Given that there are exactly 5 files involved here, we could avoid copying this code from the installer and take the same approach as in #227 - embed all the files individually and hard-code the destination path.

**Review by zaneb — COMMENTED** [review]
/lgtm

File: pkg/installer/mdsd/util.go (hunk 2 above)
Comment (rna-afk) [reply pkg/installer/mdsd/util.go:16]: I'll be using this function for the other 2 PRs as well. Probably need to add this to the dnsmasq function too which I can fix in another PR

Comment (zaneb) [reply pkg/installer/mdsd/util.go:16]: Maybe create a separate package for the data like e.g. pkg/installer/data/bootstrap then.

**Review by hlipsig — APPROVED** [review]

**Pull request merged**